src/renderer/services/wallet/balances.ts

```diff
@@ -836,19 +836,19 @@ export const createBalancesService = ({
   const chainBalances$: ChainBalances$ = FP.pipe(
     Rx.combineLatest(
       filterEnabledChains({
+        // for BTC we store `confirmed` or `all` (confirmed + unconfirmed) balances
         THOR: [thorChainBalance$, thorLedgerChainBalance$],
         MAYA: [mayaChainBalance$, mayaLedgerChainBalance$],
-        // for BTC we store `confirmed` or `all` (confirmed + unconfirmed) balances
         BTC: [btcChainBalance$, btcChainBalanceConfirmed$, btcLedgerChainBalance$, btcLedgerChainBalanceConfirmed$],
-        BCH: [bchChainBalance$, bchLedgerChainBalance$],
-        DASH: [dashBalance$, dashLedgerChainBalance$],
         ETH: [ethChainBalance$, ethLedgerChainBalance$],
+        BNB: [bnbChainBalance$, bnbLedgerChainBalance$],
         AVAX: [avaxChainBalance$, avaxLedgerChainBalance$],
```

Comment:
was any of these changes necessary ?

Reply:
Currently, the only way to change the order. It would be good if it could be linked to chain weights.